static/app/views/performance/queues/destinationSummary/destinationSummaryPage.tsx

```diff
@@ -41,6 +41,7 @@ function DestinationSummaryPage() {
   const destination = decodeScalar(query.destination);
 
   const {data} = useQueuesMetricsQuery({destination});
+  const errorRate = 1 - (data[0]?.['trace_status_rate(ok)'] ?? 0);
```

Comment:
😅 there's no `trace_status_rate(error)` for you?

Reply:
Unfortunately the trace context status has a bunch of different error states:
https://develop.sentry.dev/sdk/event-payloads/contexts/#trace-context

ie `cancelled`, `unknown`, `invalid_argument` are all possible values and are all considered errors.

I can update this in discover as a follow up!

Reply:
Makes sense! Definitely worth it to define a helper for this in Discover regardless, if possible